.github/workflows/tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         run: |
           xvfb-run -a npm run test
       - name: Upload Code Coverage
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v5
```

Comment:
`v5` is the latest major version of [`codecov/codecov-action`](https://github.com/codecov/codecov-action) at the time of this writing.